e2e/various-suite/exemplars.spec.ts

```diff
@@ -55,7 +55,7 @@ describe('Exemplars', () => {
     cy.contains(dataSourceName).scrollIntoView().should('be.visible').click();
 
     // Switch to code editor
-    cy.contains('label', 'Code').click();
+    cy.get('[data-testid="radio-button"]').contains('label', 'Code').siblings('input').click();
```

Comment:
with the new selectors, these would all become something like
```tsx
 e2e.components.RadioButton.container().filter(':contains("Code")').click();
```

Reply:
🤔 In this case, don't you click on the label? Because that's overlapped with the input and triggers a failure on the tests. Or are you clicking on the container? I'm a newbie on this so trying to understand.

Reply:
strictly speaking it's clicking the container, which then ends up being a click on the input since it lays on top.